lib/Common/Memory/SmallHeapBlockAllocator.h

```diff
@@ -173,6 +178,41 @@ SmallHeapBlockAllocator<TBlockType>::InlinedAllocImpl(Recycler * recycler, DECLS
             Assert(isSet);
         }
 #endif
+#if ENABLE_CONCURRENT_GC && ENABLE_ALLOCATIONS_DURING_CONCURRENT_SWEEP
+        if (CONFIG_FLAG_RELEASE(EnableConcurrentSweepAlloc) && recycler->AllocationsDuringconcurrentSweepTurnedOn())
```

Comment:
Lots of conditions and i wonder if they're redundant?

Reply:
I have removed the config flag check as it is now redundant.

---
In reply to: [148169494](https://github.com/Microsoft/ChakraCore/pull/4104#discussion_r148169494) [](ancestors = 148169494)